estimation/estimate_cybershake.py

```diff
@@ -319,50 +318,58 @@ def main(args):
     # Estimate HF/BB if a runs directory is specified
     if args.runs_dir is not None:
         print("Preparing HF estimation input data")
-        hf_ncores = (
+        # Have to repeat/extend the fault sim_durations to per realisation
+        r_hf_ncores = np.repeat(
             np.ones(realisations.shape[0], dtype=np.float32)
-            * estimate_wct.HF_DEFAULT_NCORES
+            * estimate_wct.HF_DEFAULT_NCORES,
+            r_counts,
         )
 
         # Get fd_count and nsub_stoch for each fault
-        fd_counts = np.asarray(
-            [
-                len(shared.get_stations(fd_statlist))
-                for fd_statlist in runs_params.fd_statlist
-            ]
+        r_fd_counts = np.repeat(
+            np.asarray(
+                [
+                    len(shared.get_stations(fd_statlist))
+                    for fd_statlist in runs_params.fd_statlist
+                ]
+            ),
+            r_counts,
         )
 
-        nsub_stochs = np.asarray(
-            [
-                srf.get_nsub_stoch(hf_slip, get_area=False)
-                for hf_slip in runs_params.hf_slip
-            ]
+        r_nsub_stochs = np.repeat(
+            np.asarray(
+                [
+                    srf.get_nsub_stoch(hf_slip, get_area=False)
+                    for hf_slip in runs_params.hf_slip
+                ]
+            ),
+            r_counts,
         )
 
-        # Have to repeat/extend the fault sim_durations to per realisation
-        r_sim_durations = np.repeat(fault_sim_durations, r_counts)
-        r_fd_counts = np.repeat(fd_counts, r_counts)
-        r_nsub_stochs = np.repeat(nsub_stochs, r_counts)
-        r_hf_nt = r_sim_durations / runs_params.hf_dt
+        # Calculate nt
+        r_hf_nt = np.repeat(fault_sim_durations, r_counts) / np.repeat(
+            runs_params.hf_dt, r_counts
+        )
```

Comment:
Is there a difference between this and np.repeat(fault_sim_duration/run_params.hf_dt, r_counts) ?